2025-08-10 17:12:15,938 INFO: Mulax Cafe startup [in C:\Users\user\Desktop\Mulaxx_coffee\mulax_cafe\app\__init__.py:278]
2025-08-10 17:12:16,014 INFO: Database connection established [in C:\Users\user\Desktop\Mulaxx_coffee\mulax_cafe\app\__init__.py:58]
2025-08-10 17:12:20,883 INFO: Registered blueprint: main [in C:\Users\user\Desktop\Mulaxx_coffee\mulax_cafe\app\__init__.py:311]
2025-08-10 17:12:20,893 INFO: Registered blueprint: auth [in C:\Users\user\Desktop\Mulaxx_coffee\mulax_cafe\app\__init__.py:311]
2025-08-10 17:12:20,902 INFO: Registered blueprint: requisitions [in C:\Users\user\Desktop\Mulaxx_coffee\mulax_cafe\app\__init__.py:311]
2025-08-10 17:12:20,913 INFO: Registered blueprint: product [in C:\Users\user\Desktop\Mulaxx_coffee\mulax_cafe\app\__init__.py:311]
2025-08-10 17:12:20,920 INFO: Registered blueprint: coffee [in C:\Users\user\Desktop\Mulaxx_coffee\mulax_cafe\app\__init__.py:311]
2025-08-10 17:12:20,925 INFO: Registered blueprint: kitchen [in C:\Users\user\Desktop\Mulaxx_coffee\mulax_cafe\app\__init__.py:311]
2025-08-10 17:12:20,935 INFO: Registered blueprint: orders [in C:\Users\user\Desktop\Mulaxx_coffee\mulax_cafe\app\__init__.py:311]
2025-08-10 17:12:20,952 INFO: Registered blueprint: tables [in C:\Users\user\Desktop\Mulaxx_coffee\mulax_cafe\app\__init__.py:311]
2025-08-10 17:12:20,965 INFO: Registered blueprint: notifications [in C:\Users\user\Desktop\Mulaxx_coffee\mulax_cafe\app\__init__.py:311]
2025-08-10 17:12:20,977 ERROR: Failed to register blueprint shifts: View function mapping is overwriting an existing endpoint function: shifts.current_shift [in C:\Users\user\Desktop\Mulaxx_coffee\mulax_cafe\app\__init__.py:313]
2025-08-10 17:12:20,981 INFO: Registered blueprint: waiters [in C:\Users\user\Desktop\Mulaxx_coffee\mulax_cafe\app\__init__.py:311]
2025-08-10 17:12:21,009 INFO: Database tables created successfully. [in C:\Users\user\Desktop\Mulaxx_coffee\mulax_cafe\app\__init__.py:122]
2025-08-10 17:12:21,298 INFO: Added API namespace: users [in C:\Users\user\Desktop\Mulaxx_coffee\mulax_cafe\app\__init__.py:341]
2025-08-10 17:12:21,302 INFO: Added API namespace: products [in C:\Users\user\Desktop\Mulaxx_coffee\mulax_cafe\app\__init__.py:341]
2025-08-10 17:12:21,310 INFO: Added API namespace: stock_movements [in C:\Users\user\Desktop\Mulaxx_coffee\mulax_cafe\app\__init__.py:341]
2025-08-10 17:12:21,316 INFO: Added API namespace: requisitions [in C:\Users\user\Desktop\Mulaxx_coffee\mulax_cafe\app\__init__.py:341]
2025-08-10 17:12:21,355 INFO: Added API namespace: coffee_sales [in C:\Users\user\Desktop\Mulaxx_coffee\mulax_cafe\app\__init__.py:341]
2025-08-10 17:12:21,373 INFO: Added API namespace: clients [in C:\Users\user\Desktop\Mulaxx_coffee\mulax_cafe\app\__init__.py:341]
2025-08-10 17:12:21,396 INFO: Added API namespace: orders [in C:\Users\user\Desktop\Mulaxx_coffee\mulax_cafe\app\__init__.py:341]
2025-08-10 17:12:21,414 INFO: Added API namespace: reports [in C:\Users\user\Desktop\Mulaxx_coffee\mulax_cafe\app\__init__.py:341]
2025-08-10 17:12:21,438 INFO: Added API namespace: shifts [in C:\Users\user\Desktop\Mulaxx_coffee\mulax_cafe\app\__init__.py:341]
2025-08-10 17:12:21,496 INFO: Scheduled tasks initialized [in C:\Users\user\Desktop\Mulaxx_coffee\mulax_cafe\app\__init__.py:248]
2025-08-10 17:12:25,135 INFO: Mulax Cafe startup [in C:\Users\user\Desktop\Mulaxx_coffee\mulax_cafe\app\__init__.py:278]
2025-08-10 17:12:25,179 INFO: Database connection established [in C:\Users\user\Desktop\Mulaxx_coffee\mulax_cafe\app\__init__.py:58]
2025-08-10 17:12:27,781 INFO: Registered blueprint: main [in C:\Users\user\Desktop\Mulaxx_coffee\mulax_cafe\app\__init__.py:311]
2025-08-10 17:12:27,783 INFO: Registered blueprint: auth [in C:\Users\user\Desktop\Mulaxx_coffee\mulax_cafe\app\__init__.py:311]
2025-08-10 17:12:27,786 INFO: Registered blueprint: requisitions [in C:\Users\user\Desktop\Mulaxx_coffee\mulax_cafe\app\__init__.py:311]
2025-08-10 17:12:27,790 INFO: Registered blueprint: product [in C:\Users\user\Desktop\Mulaxx_coffee\mulax_cafe\app\__init__.py:311]
2025-08-10 17:12:27,794 INFO: Registered blueprint: coffee [in C:\Users\user\Desktop\Mulaxx_coffee\mulax_cafe\app\__init__.py:311]
2025-08-10 17:12:27,797 INFO: Registered blueprint: kitchen [in C:\Users\user\Desktop\Mulaxx_coffee\mulax_cafe\app\__init__.py:311]
2025-08-10 17:12:27,800 INFO: Registered blueprint: orders [in C:\Users\user\Desktop\Mulaxx_coffee\mulax_cafe\app\__init__.py:311]
2025-08-10 17:12:27,805 INFO: Registered blueprint: tables [in C:\Users\user\Desktop\Mulaxx_coffee\mulax_cafe\app\__init__.py:311]
2025-08-10 17:12:27,809 INFO: Registered blueprint: notifications [in C:\Users\user\Desktop\Mulaxx_coffee\mulax_cafe\app\__init__.py:311]
2025-08-10 17:12:27,813 ERROR: Failed to register blueprint shifts: View function mapping is overwriting an existing endpoint function: shifts.current_shift [in C:\Users\user\Desktop\Mulaxx_coffee\mulax_cafe\app\__init__.py:313]
2025-08-10 17:12:27,818 INFO: Registered blueprint: waiters [in C:\Users\user\Desktop\Mulaxx_coffee\mulax_cafe\app\__init__.py:311]
2025-08-10 17:12:27,828 INFO: Database tables created successfully. [in C:\Users\user\Desktop\Mulaxx_coffee\mulax_cafe\app\__init__.py:122]
2025-08-10 17:12:27,968 INFO: Added API namespace: users [in C:\Users\user\Desktop\Mulaxx_coffee\mulax_cafe\app\__init__.py:341]
2025-08-10 17:12:27,973 INFO: Added API namespace: products [in C:\Users\user\Desktop\Mulaxx_coffee\mulax_cafe\app\__init__.py:341]
2025-08-10 17:12:27,975 INFO: Added API namespace: stock_movements [in C:\Users\user\Desktop\Mulaxx_coffee\mulax_cafe\app\__init__.py:341]
2025-08-10 17:12:27,977 INFO: Added API namespace: requisitions [in C:\Users\user\Desktop\Mulaxx_coffee\mulax_cafe\app\__init__.py:341]
2025-08-10 17:12:27,979 INFO: Added API namespace: coffee_sales [in C:\Users\user\Desktop\Mulaxx_coffee\mulax_cafe\app\__init__.py:341]
2025-08-10 17:12:27,981 INFO: Added API namespace: clients [in C:\Users\user\Desktop\Mulaxx_coffee\mulax_cafe\app\__init__.py:341]
2025-08-10 17:12:27,983 INFO: Added API namespace: orders [in C:\Users\user\Desktop\Mulaxx_coffee\mulax_cafe\app\__init__.py:341]
2025-08-10 17:12:27,985 INFO: Added API namespace: reports [in C:\Users\user\Desktop\Mulaxx_coffee\mulax_cafe\app\__init__.py:341]
2025-08-10 17:12:27,988 INFO: Added API namespace: shifts [in C:\Users\user\Desktop\Mulaxx_coffee\mulax_cafe\app\__init__.py:341]
2025-08-10 17:12:27,992 INFO: Scheduled tasks initialized [in C:\Users\user\Desktop\Mulaxx_coffee\mulax_cafe\app\__init__.py:248]
2025-08-10 17:12:28,251 INFO: Scheduler shutdown complete [in C:\Users\user\Desktop\Mulaxx_coffee\mulax_cafe\app\__init__.py:256]
2025-08-10 17:13:12,212 ERROR: NotFound: 404 Not Found: The requested URL was not found on the server. If you entered the URL manually please check your spelling and try again.
Path: /static/images/favicon.ico
Method: GET
User: Anonymous
IP: 127.0.0.1 [in C:\Users\user\Desktop\Mulaxx_coffee\mulax_cafe\app\utils\error_handlers.py:46]
Traceback (most recent call last):
  File "C:\Users\user\AppData\Local\Programs\Python\Python310\lib\site-packages\flask\app.py", line 917, in full_dispatch_request
    rv = self.dispatch_request()
  File "C:\Users\user\AppData\Local\Programs\Python\Python310\lib\site-packages\flask\app.py", line 902, in dispatch_request
    return self.ensure_sync(self.view_functions[rule.endpoint])(**view_args)  # type: ignore[no-any-return]
  File "C:\Users\user\AppData\Local\Programs\Python\Python310\lib\site-packages\flask\app.py", line 278, in <lambda>
    view_func=lambda **kw: self_ref().send_static_file(**kw),  # type: ignore # noqa: B950
  File "C:\Users\user\AppData\Local\Programs\Python\Python310\lib\site-packages\flask\app.py", line 326, in send_static_file
    return send_from_directory(
  File "C:\Users\user\AppData\Local\Programs\Python\Python310\lib\site-packages\flask\helpers.py", line 565, in send_from_directory
    return werkzeug.utils.send_from_directory(  # type: ignore[return-value]
  File "C:\Users\user\AppData\Local\Programs\Python\Python310\lib\site-packages\werkzeug\utils.py", line 575, in send_from_directory
    raise NotFound()
werkzeug.exceptions.NotFound: 404 Not Found: The requested URL was not found on the server. If you entered the URL manually please check your spelling and try again.
2025-08-10 17:13:16,215 ERROR: NotFound: 404 Not Found: The requested URL was not found on the server. If you entered the URL manually please check your spelling and try again.
Path: /static/images/favicon.ico
Method: GET
User: Anonymous
IP: 127.0.0.1 [in C:\Users\user\Desktop\Mulaxx_coffee\mulax_cafe\app\utils\error_handlers.py:46]
Traceback (most recent call last):
  File "C:\Users\user\AppData\Local\Programs\Python\Python310\lib\site-packages\flask\app.py", line 917, in full_dispatch_request
    rv = self.dispatch_request()
  File "C:\Users\user\AppData\Local\Programs\Python\Python310\lib\site-packages\flask\app.py", line 902, in dispatch_request
    return self.ensure_sync(self.view_functions[rule.endpoint])(**view_args)  # type: ignore[no-any-return]
  File "C:\Users\user\AppData\Local\Programs\Python\Python310\lib\site-packages\flask\app.py", line 278, in <lambda>
    view_func=lambda **kw: self_ref().send_static_file(**kw),  # type: ignore # noqa: B950
  File "C:\Users\user\AppData\Local\Programs\Python\Python310\lib\site-packages\flask\app.py", line 326, in send_static_file
    return send_from_directory(
  File "C:\Users\user\AppData\Local\Programs\Python\Python310\lib\site-packages\flask\helpers.py", line 565, in send_from_directory
    return werkzeug.utils.send_from_directory(  # type: ignore[return-value]
  File "C:\Users\user\AppData\Local\Programs\Python\Python310\lib\site-packages\werkzeug\utils.py", line 575, in send_from_directory
    raise NotFound()
werkzeug.exceptions.NotFound: 404 Not Found: The requested URL was not found on the server. If you entered the URL manually please check your spelling and try again.
2025-08-10 17:13:36,977 INFO: SMTP Server resolved to IP: 142.251.168.109 [in C:\Users\user\Desktop\Mulaxx_coffee\mulax_cafe\app\services\notifications.py:53]
